api/Models/RegisrtationAccountData.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace iTechArt.Hotels.Api.Models
+{
+    public class RegisrtationAccountData
+    {
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
+        [Required]
+        public string Password { get; set; }
```

Comment:
That makes code more readable, what do you think @andrei-kruhliakou-itechart?
```suggestion
        [Required]
        [EmailAddress]
        public string Email { get; set; }

        [Required]
        public string Password { get; set; }
```

Reply:
@bagatka I wouldn't make it a requirement but yes, I like the suggested style more.